tests/unit/test_configurator/conftest.py

```diff
@@ -137,6 +275,40 @@ def mock_extension(mocker):
     )
     return mock_ext
 
+@pytest.fixture(name="mock_extension")
+def mock_extension(mocker):
+    return get_extension_mocks(mocker)
+
+@pytest.fixture(name="mock_extension_status_code_already_init")
+def mock_extension_status_code_already_init(mocker):
+    return get_extension_mocks(mocker, -1)
+
+@pytest.fixture(name="mock_extension_status_code_invalid_protocol")
+def mock_extension_status_code_invalid_protocol(mocker):
+    return get_extension_mocks(mocker, 2)
```

Comment:
Great addition of these different scenarios "ok", "already_init" and "invalid_protocol"!  It's pointing out something a little subtle--we're checking the reporter initialization status as part of [reporting the Init event](https://github.com/solarwinds/apm-python/blob/51374972115712b616c8e19302e7b2f679584ed4/solarwinds_apm/configurator.py#L537), and actually ignoring the return code from the report Init event `sendStatus` call.